modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/TestBuilders.java

```diff
@@ -739,14 +739,6 @@ public TestCluster build() {
                     CatalogTestUtils.createCatalogManagerWithTestUpdateLog(clusterName, clock)
             );
 
```

Comment:
do we still need changes in this file?

Reply:
I think no, let's wait for tests run to prove it. Only no-op validator is the change now.